kadena-beta/docs/Kadena-README.md

```diff
@@ -544,14 +556,14 @@ etc.
 
 #### Sample Usage: Viewing the Performance Monitor
 
-Each kadena node, while running, will host a performance monitor at the URL `<nodeId.host>:<nodeId.port>/monitor`.
+Each Kuro node, while running, will host a performance monitor at the URL `<nodeId.host>:<nodeId.port>/monitor`.
 
 #### Sample Usage: Running Pact TodoMVC
 
-This repo also bundles the [Pact TodoMVC](github.com/kadena-io/pact-todomvc). Each Kadena node will host the frontend at `<nodeId.host>:<nodeId.port>/todomvc`. To initialized the `todomvc`:
+This repo also bundles the [Pact TodoMVC](github.com/kadena-io/pact-todomvc). Each Kuro node will host the frontend at `<nodeId.host>:<nodeId.port>/todomvc`. To initialized the `todomvc`:
```

Comment:
Let's ditch this line as I want to get rid of the TodoMVC distro in here

Reply:
Should I go ahead and delete the todomvc sub-directory here too?

Reply:
Also do you mean get rid of the entire `#### Sample Usage: Running Pact TodoMVC` section or just the line `This repo also bundles the [Pact TodoMVC](github.com/kadena-io/pact-todomvc).`?

Reply:
Yeah let's get rid of all of it